ci-operator/config/openshift/installer/master.yaml

```diff
@@ -1,44 +1,21 @@
 base_images:
   base:
     cluster: https://api.ci.openshift.org
-    name: origin-v4.0
-    namespace: openshift
-    tag: base
-  base-smoke:
-    cluster: https://api.ci.openshift.org
-    namespace: openshift
     name: release
-    tag: bazel
-binary_build_commands: go build ./installer/cmd/tectonic
+    namespace: openshift
+    tag: golang-1.10
+binary_build_commands: hack/build.sh && hack/get-terraform.sh
 canonical_go_repository: github.com/openshift/installer
 images:
-- dockerfile_path: images/tectonic-installer/Dockerfile.ci
-  from: base
+- from: base
   inputs:
     bin:
       paths:
       - destination_dir: .
-        source_path: /go/src/github.com/openshift/installer/tectonic
-    root:
-      as:
-      - build
-  to: installer
-- dockerfile_path: images/tectonic-installer/Dockerfile.ci
-  optional: true
-  from: base-smoke
-  inputs:
-    bin:
-      paths:
+        source_path: /go/src/github.com/openshift/installer/bin/openshift-install
       - destination_dir: .
-        source_path: /go/src/github.com/openshift/installer/tectonic
-    root:
-      as:
-      - build
-  to: installer-bazel
-- dockerfile_path: images/installer-origin-release/Dockerfile.ci
-  optional: true
-  from: installer-bazel
-  to: installer-smoke
+        source_path: /go/src/github.com/openshift/installer/bin/terraform
+  to: installer
```

Comment:
Is the replacement dockerfile centos based and using the right tooling?

Reply:
> Is the replacement dockerfile centos based and using the right tooling?

There is no replacement `Dockerfile`.  openshift/release:golang-1.10 has what we need for `hack/build.sh && hack/get-terraform.sh`, and I was hoping to build the `:installer` image with just that [here][1].  But I don't know what I'm doing with ci-operator; maybe that's not how it works?

[1]: https://github.com/openshift/release/blob/37c66462b2d5671f2a44c2067e6e1a7270cef907/ci-operator/config/openshift/installer/master.yaml#L1-L18

Reply:
> There is no replacement `Dockerfile`.

This was a dead end.  I've pushed a replacement `Dockerfile` with openshift/installer#343, but it's based on `scratch`, not CentOS.  Do we need CentOS for something?

Reply:
ART team recommends this, may be outright required, I'm not sure. (https://mojo.redhat.com/docs/DOC-1179058#jive_content_id_Images)

I also quickly noticed I can't exec in with /bin/bash, which is awfully handy for debugging. Actually in the way we're probably going to use this in Hive we probably need a more substantial base layer.

Reply:
> I also quickly noticed I can't exec in with /bin/bash, which is awfully handy for debugging. 

The `Dockerfile` is stuffing the binaries into `scratch` (for folks who just need the binaries), but I couldn't figure out the right phrasing for that here (lucky for you ;), so we're currently [using the base image][1] (defined [here][2]) as the base layer that gets the binaries added.  I chat about this a bit in the 3f2f01c43 commit message if you want more details.

[1]: https://github.com/openshift/release/blob/3f2f01c43b51725b685621a95a3acc2115ddfb47/ci-operator/config/openshift/installer/master.yaml#L10
[2]: https://github.com/openshift/release/blob/3f2f01c43b51725b685621a95a3acc2115ddfb47/ci-operator/config/openshift/installer/master.yaml#L1-L6

Reply:
Ah so your published CI output is quite different from your scratch Dockerfile.ci if you built it yourself? As long as the published image is CentOS/RHEL based I think we'll have what we need.

You could also consider multiple Dockerfiles if need be. I can hack my own for now until this is getting publishing.